app/components/Header.tsx

```diff
@@ -82,6 +82,13 @@ export function Header({ email }: { email?: string }) {
                       >
                         Profile
                       </Link>,
+                      <Link
+                        key="permissions"
+                        to="/user/permissions"
```

Comment:
We need a more descriptive name for this page, because "Credentials" and "Permissions" sound too similar.

Reply:
Sounds good.